src/actions/channels.test.js

```diff
@@ -2278,4 +2278,52 @@ describe('Actions.Channels', () => {
 
         assert.equal(error, null);
     });
+
+    it('getChannelModerations', async () => {
+        const channelID = 'cid10000000000000000000000';
+
+        nock(Client4.getBaseRoute())
+        .get(`/channels/${channelID}/moderations`).
+        reply(200,  [{
+                "name": "create_posts",
+                "roles": {
+                    "members": true,
+                    "guests": false,
+                },
+            }]
+        );
+
+        const {error} = await Actions.getChannelModerations(channelID)(store.dispatch, store.getState);
```

Comment:
Actions should actually be passed into `dispatch` instead of called directly. I'm going to create a ticket to go do a mass refactor of these since we've been slowly changing that as we come across it, but that hasn't stopped more from being added.

```suggestion
        const {error} = await store.dispatch(Actions.getChannelModerations(channelID));
```